### DISCUSSION TITLE
Humidity Forecast

### DISCUSSION THREAD

**Discussion by tltoburen** [weather-gov/api#276] [Q&A]
Hi, new here... I need to pull a 5-day forecast into Excel for some analysis I'm doing.

I have the hourly forecast for temperature in and parsing correctly, but I need all three of the following:
Temperature
Relative Humidity
Local Station Barometric Pressure

Only temperature is in the API response, but I see all the information in graphic versions on the weather.gov site.  How can I add the humidity and pressure forecasts into Excel?

Thanks!
-T

**Comment by scadergit** [comment]
Unless I'm miss-understanding, you're mixing two, separate services: forecasts and observations. You can find the raw forecast data on the gridpoint endpoint itself, e.g.: https://api.weather.gov/gridpoints/RAH/68,58 (no /forecast at the end of the path). 

You'll then need to use the observation station endpoints for current observations.

**Comment by tltoburen** [comment]
I'm looking for the forecasts for humidity and forecast pressures as well as temperature.  Not current or historical observations.
